src/components/modals/AddAccounts/steps/03-step-import.js

```diff
@@ -353,25 +366,34 @@ export const StepImportFooter = ({
     <Fragment>
       {currency && <CurrencyBadge mr="auto" currency={currency} />}
       {scanStatus === 'error' && (
-        <Button mr={2} onClick={() => setScanStatus('scanning')}>
-          {t('app:common.retry')}
-        </Button>
+        <Fragment>
+          <Button
+            mr={2}
+            outlineGrey
+            onClick={() => customerSupport()}
+            style={{ borderRadius: '4px' }}
+          >
+            {t('app:common.support')}
+          </Button>
```

Comment:
actually, I think this should be made generic to be a `ContactButton` that would just propagate props to Button but the onClick and children would be implemented

```
          <ContactButton
            mr={2}
            outlineGrey
            style={{ borderRadius: '4px' }}
          />
```

this would be beneficial not only to sharing more code but also the day we'll refactor this to trigger a modal

Reply:
👍